### DISCUSSION TITLE
echarts柱状图添加

### DISCUSSION THREAD

**Discussion by MangMax** [zouyaoji/vue-cesium#380] [Q&A] [answered]
请教一下echarts的柱状图如何添加进来，柱状图好像也不能设置coordinateSystem属性，顺便问一下官网这个涟漪效果的图片应该怎么制作

**Comment by zouyaoji** [comment]
柱状图你直接搞个div叠加到球上就OK了吗？

图片网上搞得~

**Reply by MangMax** [reply]
好的，谢谢，还想问一下我更改option，组件好像不会跟着变化，我怎样实现动态更新数据呢，这个组件内部是不是应该有watch图表option的操作来实现实时更新

**Reply by MangMax** [reply]
目前我是通过reload来实现刷新的，不知道是不是正确的方法是

**Reply by zouyaoji** [reply, marked as answer]
要自己watch的
